### PR TITLE
Icu 5991 UI add worker filter form field for vault type credential stores

### PR DESCRIPTION
**Pull request by DhariniJeeva**
:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-5991)
 
[admin-ui](https://boundary-ui-git-icu-5991-ui-add-worker-filter-9fe9ca-hashicorp.vercel.app/scopes/s_1sheljmjgm/credential-stores/cs_ucau082vov)

doc link is added [here](https://github.com/hashicorp/boundary/pull/2447)

### CODE REVIEW

**Review by cameronperera — APPROVED** [review]
LGTM!

File: addons/core/translations/resources/en-us.yaml [hunk 1]
```diff
@@ -373,6 +373,9 @@ credential-store:
       help: A PEM-encoded private key matching the client certificate from Client Certificate.
     client_certificate_key_hmac:
       label: Client Certificate Key HMAC
+    worker_filter:
```
Comment (ZedLi) [inline addons/core/translations/resources/en-us.yaml:376]: It looks like we have an identical translation in the `forms` [translations](https://github.com/hashicorp/boundary-ui/blob/main/addons/core/translations/form/en-us.yaml#L56), should we just use that instead?

**Review by ZedLi — APPROVED** [review]
LGTM, just left one comment

File: addons/core/translations/resources/en-us.yaml (hunk 1 above)
Comment (DhariniJeeva) [reply addons/core/translations/resources/en-us.yaml]: Done!

**Pull request merged**